### ISSUE TITLE
When no name is specified in mdx file, it throws Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by fc** [pedronauck/docz#59]
## Bug Report

**To Reproduce**

Create an MDX file with no name specified.
It will generate the error:

```
$ docz dev
TypeError: Cannot read property 'name' of undefined
    at new Entry (.../frontend/node_modules/docz-core/dist/index.js:1:5189)
    at Promise.all.s.filter.map (.../frontend/node_modules/docz-core/dist/index.js:1:7654)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:746:11)
    at startup (internal/bootstrap/node.js:238:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:572:3)
```

**Expected behavior**

Provide a helpful error message that $path/to/file.mdx does not contain a name.

It probably occurs at this line but couldn't find out how to run docz without using the `dist` version to validate:
https://github.com/pedronauck/docz/blob/master/packages/docz-core/src/Entry.ts#L98

**Comment by vivek12345** [comment]
@pedronauck I would like to take this up.The solution I am thinking here is throwing an error with the message of $path/to/file.mdx does not contain a name.

**Comment by vivek12345** [comment]
@pedronauck While trying to do a yarn commit I am facing this error
```
stderr: 'ERROR: Task not found: "/Users/viveknayyar/vivek/docz/packages/docz-core/src/Entry.ts"
```

**Comment by vivek12345** [comment]
@pedronauck Can you help here in this yarn commit issue that I am facing.

**Comment by pedronauck** [comment]
Do you trying to build packages to make some pull request @vivek12345? Which version of node you're using?

**Comment by vivek12345** [comment]
Node version is v9.0.0

**Comment by vivek12345** [comment]
I did those above mentioned changes inside docz-core/src/Entry.js
After that did a yarn build in docz-core folder.

Finally git add and yarn commit

**Comment by vivek12345** [comment]
Thank you @marceloavf for your help. I have raised a PR for the same.

**Comment by pedronauck** [comment]
## 🎉Fixed and released in the new [v0.4.0](https://github.com/pedronauck/docz/releases/tag/v0.4.0)
